shared/api/models/DateModel.ts

```diff
@@ -90,9 +96,18 @@ class DateModel {
     return localRecurrenceRule
       .between(minDate, maxDate, true, (_, index) => index < count)
       .map(offsetDate => {
-        const actualDate = DateTime.fromJSDate(offsetDate).plus({ minutes: offsetDate.getTimezoneOffset() })
+        const actualDate = DateTime.fromJSDate(offsetDate, { zone: 'local' }).toUTC()
```

Comment:
seems like `getTimezoneOffset` is causing the issue, can't we just set the zone to `local` and convert it to UTC. Just changing this line seems like fixes the issue :thinking: We could also adjust the tests accordingly...

Reply:
@michellegraf and I just had a look at the tests and couldn't find a spot where the time differed, but we also didn't manage to check all of them. It might work because we're all in the German time zone. Maybe adapt the tests and then run them with a different time zone on your laptop?